src/proof/alf/alf_printer.cpp

```diff
@@ -202,6 +202,12 @@ bool AlfPrinter::isHandled(const ProofNode* pfn) const
              || k == Kind::STRING_INDEXOF || k == Kind::STRING_IN_REGEXP;
     }
     break;
+    case ProofRule::ALPHA_EQUIV:
+    {
+      // only works in general if not using fresh binders
+      return !options().parser.freshBinders;
```

Comment:
IIUC, the idea of something not being handled is that it'll lead to a *valid*, but admitted (a proof hole) step. Is this gonna be the case here or will this be an invalid step actually?

Reply:
This was an attempt to fix the CI issues, I'm going to revert this.

There is a more general issue with quantifiers proofs in the current state of this PR which I need to address.